DATA/MISSIONS/npcships.ini

```diff
@@ -120,656 +390,761 @@ npc_class = lawful, class_cruiser, d1, d2, d3, d4, d5, d6, d7, d8, d9, d10, d11,
 ;; ZIVIL Starflyer
 
 [NPCShipArch]
-nickname = npc_co_ge_fighter_d1-2
+nickname = npc_co_ge_fighter_d1_a
+;;nickname = npc_co_ge_fighter_d1-2

```

Comment:
Is there a reason to keep those old names as comment?

Reply:
These are old (not used anymore) entries, i used them to compare with old and new Npc Balancing, i will create an extra branch for "clean up"